tests/fixtures/application_fixtures.py

```diff
@@ -914,3 +918,16 @@ def sample_data():
             ["1", "a", [["z", "a", "a"]]],
         ],  # Lists elems are different types, not officially supported
     }
+
+
+@pytest.fixture(scope="session")
+def mongo_example_db() -> Generator:
+    """Return a connection to the MongoDB example DB"""
+    uri = "mongodb://mongo_user:mongo_pass@mongodb_example/mongo_test"
+
+    client = MongoClient(uri, serverSelectionTimeoutMS=5000)
+    logger.debug(f"Connecting to MongoDB example database at: {uri}")
+    # Setup above...
+    yield client
+    # Teardown below...
+    client.close()
```

Comment:
We already have a fixture `integration_mongodb_connector` for this.

Reply:
ah I was just moving the existing fixture to a place where I could use it more easily, but I'm not sure what changes were made in the fixture PR so I'll take a closer look and get this cleaned up.

Reply:
Fixture removed in favor of integration_mongodb_connector